services/index.html

```diff
@@ -133,23 +133,23 @@ <h3 class="panel-title">{% t services-training_coaching.title %}</h3>
 							<div class="service">
 									<a class="button-with-bg-image"   href="/clients#finance">
 											<img class="img-responsive" src="/assets/img/custom/services/service-bg-2.png" alt="">
-											<span>FINANCE</span>
+											<span>{% t services-section_2.finance %}</span>
 									</a>
 							</div>
 					</div>
 					<div class="col-md-4">
 							<div class="service">
 									<a class="button-with-bg-image"  href="/clients#retail">
 											<img class="img-responsive" src="/assets/img/custom/services/service-bg-2.png" alt="">
-											<span>RETAIL</span>
+											<span>{% t services-section_2.retail %}</span>
```

Comment:
The Retail and Media buttons don't look correct due to the length of the text. Would anyone mind to have a look at the styling?
<img width="947" alt="screen shot 2017-01-06 at 09 24 02" src="https://cloud.githubusercontent.com/assets/5873601/21713540/6263243a-d3f2-11e6-8d58-28fe8e07d48d.png">